### ISSUE TITLE
Preferences lost every chrome restart?

### ISSUE THREAD

**Issue by paulirish** [giuseppeg/refined-twitter-lite#9]
My prefs:

```js
RefinedTwitterLite.setFeatures({ singleColumn: false, enforceLatestTweets: false , hideExplore: true })
```

all i know is i get the "latest tweets" timeline every once in a while, despite having executed that line in my console a day or so ago. It just happened again today, and since it appears there hasn't been a republish at CWS.. I'm guessing its related to me restarting the browser (which i don't do too often).

(I don't know for sure that the singleColumn/hideExplore items were also ignored... )

**Comment by paulirish** [comment]
i just restarted chrome to doublecheck. and yes, i also don't have singleColumn or hideExplore

also lol, this pizza subreddit link. 🤣 

I'm guessing its the localstorage merging bit where my `false` is overwritten with the default's `true`..

Edit: i tried stepping through with breakpoints but the problem didnt repro that time. :/

**Comment by giuseppeg** [comment]
Is it possible that you had other open tabs and this bad guy reset your preferences? 
https://github.com/giuseppeg/refined-twitter-lite/blob/3178ec9ffb9a07e598f80fc8e580d7f3bfec24ba/index.user.js#L805-L807

> also lol, this pizza subreddit link. 🤣 

<3 :D

**Comment by benthroop** [comment]
Yeah I'm uninstalling because of this. Looking forward to a fix if you get to it! Lovely tool otherwise!
